backend/game_handler.py

```diff
@@ -3,6 +3,61 @@
 from threading import Lock
 
 
+class GameHandler:
+
+    def __init__(
+        self,
+        name: str = "New Lobby",
+        average: float = 30,
+        standard_deviation: float = 10,
+        minimum_time: float = 3,
+        starting_multiplicator: float = 0.75,
+        multiplicator_coef: float = 0.01,
+    ):
+        self.name = name
+        self.average = average
+        self.std = standard_deviation
+        self.min_time = minimum_time
+
+        self.starting_multiplicator = starting_multiplicator
+        self.multiplicator_coef = multiplicator_coef
+
+        self.players = {}  # ws : name
+
+        self.cash = {}  # ws : cash
+
+        # For simultaneous access
+        self.mutex = Lock()
+
+    # only for some testing
+    def test_values(self) -> tuple:
+        duration = max(self.min_time, np.random.normal(self.average, self.std))
+        time_array = np.linspace(0, duration)
+        multiplicator_array = self.starting_multiplicator + np.exp(
+            self.multiplicator_coef * time_array
+        )
+        return duration, multiplicator_array
+
+    def get_game(self, name: str = "New Game"):
+
+        return Game(
+            name,
+            starting_multiplicator=self.starting_multiplicator,
+            multiplicator_coef=self.multiplicator_coef,
+            average=self.average,
+            standard_deviation=self.std,
+            minimum_time=self.min_time,
+            game_handler_parent=self,
+        )
+
+    def join(self, ws, name):
+        with self.mutex:
+            self.players[ws] = name
+
+        if ws not in self.cash:
+            self.cash[ws] = 1000
```

Comment:
this should also be in the mutex lock since it falls under the same shared memory pattern